modules/common-user-vpc/variables.tf

```diff
@@ -18,12 +18,19 @@ variable "subnet_tags" {
   default     = {}
 }
 
-variable "allow_list" {
+variable "ingress_allow_list" {
   type        = "list"
-  description = "list of CIDRs we allow to access the infrastructure"
+  description = "list of CIDR blocks we allow to access the infrastructure"
   default     = []
 }
 
+variable "egress_allow_list" {
+  type        = "list"
+  description = "List of CIDR blocks we allow the infrastructure to access"
+  default     = ["0.0.0.0/0"]
```

Comment:
default here for if an end user pulls out the submodules and builds something but still wants open egress.